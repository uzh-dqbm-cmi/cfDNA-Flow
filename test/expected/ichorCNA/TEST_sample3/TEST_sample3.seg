sample	chr	start	end	event	copy.number	bins	median
TEST_sample3	1	1000001	248000000	NEUT	2	247	0.000613145392377744
TEST_sample3	2	1000001	242000000	NEUT	2	241	-0.00262137688929523
TEST_sample3	3	1000001	198000000	NEUT	2	197	0.000913225960882563
TEST_sample3	4	2000001	189000000	NEUT	2	187	-0.000235159291664848
TEST_sample3	5	1000001	181000000	NEUT	2	180	-0.00247422199512458
TEST_sample3	6	1000001	170000000	NEUT	2	169	-0.0116512051502193
TEST_sample3	7	1000001	159000000	NEUT	2	158	0.00745151988619165
TEST_sample3	8	1000001	145000000	NEUT	2	144	0.0613347369683535
TEST_sample3	9	1000001	138000000	NEUT	2	137	0.000113972908230232
TEST_sample3	10	1000001	133000000	NEUT	2	132	-0.00791318229927602
TEST_sample3	11	1000001	135000000	NEUT	2	134	0.0129929115382461
TEST_sample3	12	1000001	132000000	NEUT	2	131	0.00589485193707233
TEST_sample3	13	20000001	113000000	NEUT	2	93	-0.0169184847445049
TEST_sample3	14	20000001	105000000	NEUT	2	85	0.00157830837473238
TEST_sample3	15	25000001	101000000	NEUT	2	76	-0.0112544640139748
TEST_sample3	16	1000001	90000000	NEUT	2	89	0.0193104731222986
TEST_sample3	17	1000001	83000000	NEUT	2	82	-0.00882063747999508
TEST_sample3	18	1000001	80000000	NEUT	2	79	-0.00866771180566091
TEST_sample3	19	1000001	58000000	NEUT	2	57	-0.0221395980974821
TEST_sample3	20	1000001	64000000	NEUT	2	63	0.0088004397494226
TEST_sample3	21	14000001	46000000	NEUT	2	32	-0.0217947939827096
TEST_sample3	22	19000001	49000000	NEUT	2	30	0.00639690981130164
TEST_sample3	X	4000001	154000000	NEUT	2	150	0.00438290753422059
