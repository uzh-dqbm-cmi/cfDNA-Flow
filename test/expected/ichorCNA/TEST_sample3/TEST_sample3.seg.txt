ID	chrom	start	end	num.mark	seg.median.logR	copy.number	call	subclone.status	logR_Copy_Number	Corrected_Copy_Number	Corrected_Call
TEST_sample3	1	1000001	248000000	247	0.000613145392377744	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	2	1000001	242000000	241	-0.00262137688929523	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	3	1000001	198000000	197	0.000913225960882563	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	4	2000001	189000000	187	-0.000235159291664848	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	5	1000001	181000000	180	-0.00247422199512458	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	6	1000001	170000000	169	-0.0116512051502193	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	7	1000001	159000000	158	0.00745151988619165	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	8	1000001	145000000	144	0.0613347369683535	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	9	1000001	138000000	137	0.000113972908230232	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	10	1000001	133000000	132	-0.00791318229927602	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	11	1000001	135000000	134	0.0129929115382461	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	12	1000001	132000000	131	0.00589485193707233	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	13	20000001	113000000	93	-0.0169184847445049	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	14	20000001	105000000	85	0.00157830837473238	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	15	25000001	101000000	76	-0.0112544640139748	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	16	1000001	90000000	89	0.0193104731222986	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	17	1000001	83000000	82	-0.00882063747999508	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	18	1000001	80000000	79	-0.00866771180566091	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	19	1000001	58000000	57	-0.0221395980974821	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	20	1000001	64000000	63	0.0088004397494226	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	21	14000001	46000000	32	-0.0217947939827096	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample3	22	19000001	49000000	30	0.00639690981130164	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample3	X	4000001	154000000	150	0.00438290753422059	2	NEUT	FALSE	Inf	2	NEUT
